### PR TITLE
Use appropriate database

### PR DESCRIPTION
**Pull request by esoergel**
This is a bunch of stuff that would be broken by moving domains to another db.

### CODE REVIEW

File: corehq/apps/cleanup/management/commands/migrate_repeat_records.py [hunk 1]
```diff
@@ -1,28 +0,0 @@
...
-        doc_ids = get_doc_ids_in_domain_by_type(domain, 'RepeatRecord', old_db)
-        count = len(doc_ids)
-        chunksize = 250
-
-        for i, docs in enumerate(chunked(iter_docs(old_db, doc_ids, chunksize), chunksize)):
-            for doc in docs:
-                if '_rev' in doc:
-                    del doc['_rev']
-            new_db.bulk_save(docs, new_edits=False)
-            print 'checked %s / %s' % (i * chunksize, count)
```
Comment (esoergel) [inline corehq/apps/cleanup/management/commands/migrate_repeat_records.py]: This is an old migration command, so I assume it's safe to delete.  @czue can you confirm?

Comment (czue) [inline corehq/apps/cleanup/management/commands/migrate_repeat_records.py]: yep, this is just a one-off poor-man's implementation of doctypemigrations

**Comment by esoergel** [comment]
@dannyroberts @millerdev FYI

File: corehq/apps/cleanup/management/commands/migrate_repeat_records.py (hunk 1 above)
Comment (esoergel) [inline corehq/apps/cleanup/management/commands/migrate_repeat_records.py]: awesome, thanks!

File: corehq/apps/cleanup/management/commands/repeater_times.py [hunk 2]
```diff
@@ -13,7 +12,7 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
         domain = args[0]
-        db = Domain.get_db()
+        db = RepeatRecord.get_db()
```
Comment (dannyroberts) [inline corehq/apps/cleanup/management/commands/repeater_times.py]: `receiverwrapper` has had its own DB for a long time now, so this managemtn command can't have worked for a long time. Given that, I'd opt to just delete it!

File: corehq/apps/cleanup/management/commands/couch_migrate.py [hunk 3]
```diff
@@ -49,7 +49,7 @@ def handle(self, *args, **options):
 def _copy(config):
     # unfortunately the only couch view we have for this needs to go by domain
     # will be a bit slow
-    database = Domain.get_db()
+    database = get_db()
```
Comment (dannyroberts) [inline corehq/apps/cleanup/management/commands/couch_migrate.py]: Looks like this this management command is deprecated in favor of `doctypemigrations`'s `run_doctype_migration`.

Comment (dannyroberts) [inline corehq/apps/cleanup/management/commands/couch_migrate.py]: @czue fine to delete this command?

Comment (czue) [inline corehq/apps/cleanup/management/commands/couch_migrate.py]: yep, seems good

**Comment by esoergel** [comment]
Yay, deleting stuff!  Updated.

**Comment by esoergel** [comment]
(cancelling build)

**Comment by dannyroberts** [comment]
:goberserk:

**Pull request merged**